### DISCUSSION TITLE
debian - installazione e primo accesso

### DISCUSSION THREAD

**Discussion by Paol-O** [indomus/forum#3177] [GENERALE - Discussioni generiche]
Ciao a tutti, ho proceduto all'installazione di Linux Debian 12 (“Bookworm”) su un NiPoGi Mini PC N97
L'installazione è stata completata con successo (sebbene abbia dovuto farla tramite hotspot da telefono, per problemi temporanei di linea internet), ma al momento del riavvio, quando il sistema chiede nome utente e password, posso inserire il nome utente, ma alla successiva riga di comando (password) la tastiera non risponde con nessun tasto, eccezion fatta per "invio", cui segue il messaggio "incorrect".

Sono sicuro di aver inserito la corretta tipologia di tastiera (tant'è che nella prima riga, del nome, tutti i caratteri rispondono e corrispondono), ho provveduto a rifare l'installazione, ma il problema persiste.

**Comment by snakuzzo** [comment]
Guarda che è del tutto normale.
Per questioni di sicurezza non ti fa vedere nulla quando digiti la password, ma in realtà lo stai facendo :)

**Reply by Paol-O** [reply]
ti ringrazio per la risposta, mi hanno spiegato altrove che linux non mette gli * per le password, non lo sapevo.

grazie ancora

**Reply by snakuzzo** [reply]
Te l'ho spiegato anche io in realtà :D

**Reply by Paol-O** [reply]
si, hai ragione, ma avevo letto prima del tuo altri commenti su facebook alla mia analoga richiesta di aiuto.
ti ringrazio ancora
